UnityProject/Assets/Scripts/Switches/LightSwitchV2.cs

```diff
@@ -50,6 +50,11 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(HandApply interaction)
 		{
+			if (!isServer) 
```

Comment:
ServerPerformInteraction is only ever called on server

Reply:
You're right. Another shotgun moment. Never saw that log ( or those other unnecessary logs ) ever pop up in my tests so these isServer checks were only the chopping block either way.